README.md

```diff
@@ -127,6 +127,11 @@ the project are:
   GIL][a-parallelism], and novel [memory management and garbage collection
   techniques][a-memory-management].
 
+### Minimum Supported Rust Version
```

Comment:
Artichoke already documents its Rust toolchain requirements in `BUILD.md`. I'd rather keep the main readme devoted to the pitch instead of build details. Can we remove this change?

Do you think the docs here are sufficient?

https://github.com/artichoke/artichoke/blob/7924a8eb304deb8d82c8c434590e7120943d97cf/BUILD.md#rust-toolchain